README.md

```diff
@@ -92,6 +92,7 @@ Supported indicates if a specific Ruby version / Implementation is actively supp
 | cRUBY/MRI      | 3.2            | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |
 | cRUBY/MRI      | 3.3            | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |
 | cRUBY/MRI      | 3.4            | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |
+| cRUBY/MRI      | 4.0            | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |
```

Comment:
We did not add any ruby 4.0 mutations and or did not port unparser yet to cover 4.0 so we have to leacce them as TBD.

Reply:
syntax and mutations are TBD.